### PR TITLE
draft: windows support

### PR DESCRIPTION
**Pull request by chiefbiiko**
been diggin into windows wormholes...

- [x] `cargo mobile init` as administrator otherwise symlinking fails
- [x] `cargo android build`
- [ ] `cargo android run` - installing the apk seems 2 work - running the app fails on my 35€ zte blade l130
- [ ] `cargo android open`

### CODE REVIEW

**Comment by francesca64** [comment]
Thanks so much for tackling this! It's a huge help.

It looks like creating symlinks without admin is possible: https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/

That was formally released April 2017, which seems reasonably long ago to me.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,14 @@ Whenever you want to update:
 cargo mobile update
 ```
 
+### Windows
+
+We need to tell `cargo` to not install the `cargo-apple` binary on Windows.
```
Comment (francesca64) [inline README.md:42]: There's a solution for this in #5 that I'm happy with, so you can just rebase onto that once it's merged.

**Comment by chiefbiiko** [comment]
> It looks like creating symlinks without admin is possible: https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/
 
the link says unelevated `mklink` is possible if ["developer mode" is enabled](https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development)
havent heard of win dev mode yet - anyways, imho this dev mode hurdle is coequal to the admin hurdle
i can investigate the `CreateSymbolicLink` API but am kinda sure it is what Rust's `std::os::windows::fs` uses and what is currently used in this draft pr

File: README.md (hunk 1 above)
Comment (chiefbiiko) [reply README.md:42]: o great, this was bugging me

**Comment by chiefbiiko** [comment]
i am pretty tight on time atm
anyways i would still love to help push cargo-mobile 2 windows
maybe we can outline priorities - `cargo android run` prio high? - `cargo android open` prio low? ...?

**Comment by francesca64** [comment]
Since it looks like Windows doesn't make symlinks easy, we can just copy the files instead. There wouldn't really be any downsides.

`cargo android run`, `cargo android build`, and `cargo android check` are the big things for minimal Windows support. `cargo mobile init` needs to work too, of course.

`cargo mobile open` and `cargo android open` are definitely lower priorities.

**Comment by chiefbiiko** [comment]
closing in vain

**Pull request closed without merging**